[2024-10-25T08:40:50.876+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-25T08:40:50.900+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: indices_argentina_v02.load scheduled__2024-10-23T00:00:00+00:00 [queued]>
[2024-10-25T08:40:50.909+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: indices_argentina_v02.load scheduled__2024-10-23T00:00:00+00:00 [queued]>
[2024-10-25T08:40:50.910+0000] {taskinstance.py:2865} INFO - Starting attempt 8 of 13
[2024-10-25T08:40:50.926+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load> on 2024-10-23 00:00:00+00:00
[2024-10-25T08:40:50.932+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-25T08:40:50.935+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=23435) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-25T08:40:50.935+0000] {standard_task_runner.py:72} INFO - Started process 23542 to run task
[2024-10-25T08:40:50.936+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'indices_argentina_v02', 'load', 'scheduled__2024-10-23T00:00:00+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp7xmkfkqc']
[2024-10-25T08:40:50.938+0000] {standard_task_runner.py:105} INFO - Job 134: Subtask load
[2024-10-25T08:40:50.989+0000] {task_command.py:467} INFO - Running <TaskInstance: indices_argentina_v02.load scheduled__2024-10-23T00:00:00+00:00 [running]> on host d9df5aecafce
[2024-10-25T08:40:51.255+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rolo' AIRFLOW_CTX_DAG_ID='indices_argentina_v02' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-10-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-23T00:00:00+00:00'
[2024-10-25T08:40:51.256+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-25T08:40:51.275+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-25T08:40:51.290+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-25T08:40:51.290+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=indices_argentina_v02, task_id=load, run_id=scheduled__2024-10-23T00:00:00+00:00, execution_date=20241023T000000, start_date=20241025T084050, end_date=20241025T084051
[2024-10-25T08:40:51.351+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-25T08:40:51.369+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-25T08:40:51.371+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
